.circleci/config.yml

```diff
@@ -1,67 +1,19 @@
 version: 2.1
 
-orbs:
-  codecov: codecov/codecov@3.2.2
-
-jobs:
-  tests:
-    docker:
-      - image: 'cimg/python:3.10'
-        environment:
-          TOX_POSARGS: ''
-      - image: 'docker.elastic.co/elasticsearch/elasticsearch:7.14.0'
-        name: search
-        environment:
-          discovery.type: single-node
-          ES_JAVA_OPTS: -Xms750m -Xmx750m
-    steps:
-      - checkout
-      - run: git submodule sync
-      - run: git submodule update --init
-      - run: pip install --user 'tox<5'
-      - run: tox -e py310
-      - codecov/upload
+setup: true
 
-  tests-embedapi:
-    docker:
-      - image: 'cimg/python:3.10'
-    steps:
-      - checkout
-      - run: git submodule sync
-      - run: git submodule update --init
-      - run: pip install --user 'tox<4'
-      - run: tox -c tox.embedapi.ini
-
-  checks:
-    docker:
-      - image: 'cimg/python:3.10'
-        environment:
-          NODE_VERSION: 10.17.0
-    steps:
-      - checkout
-      - run: git submodule sync
-      - run: git submodule update --init
-      - run: git fetch origin main  # needed for comparisson in pre-commit
-      - run: git branch -f --track main origin/main  # needed for comparisson in pre-commit
-      - run: pip install --user 'tox<5'
-      - run: tox -e pre-commit
-      - run: tox -e migrations
-      - run: tox -e lint
-      - run: scripts/circle/install_node.sh
-      - run:
-          name: Add node to the path
-          command: |
-            echo 'export PATH=~/.nvm/versions/node/v${NODE_VERSION}/bin:$PATH' >> $BASH_ENV
-            source $BASH_ENV
-      - run: tox -e eslint
+orbs:
+  path-filtering: circleci/path-filtering@0.1.3
 
 workflows:
-  version: 2
-  test:
+  generate-config:
     jobs:
-      - checks
-      - tests
-      - tests-embedapi:
-          requires:
-            - checks
-            - tests
+      - path-filtering/filter:
+          base-revision: main
```

Comment:
When we target branches different from `main`, there might be some false positives. I've opened up a feature request: https://github.com/CircleCI-Public/path-filtering-orb/issues/65